build.rs

```diff
@@ -1,40 +1,13 @@
-use std::env;
-
+#[cfg(all(unix, not(target_os = "macos")))]
 fn main() {
 
-    // required libraries
-    // https://www.infradead.org/openconnect/building.html
-
-    let target = &env::var("TARGET").unwrap();
-    match target.as_str() {
-        "x86_64-apple-darwin" => {
-            // brew install gettext
-            println!("cargo:rustc-link-search=/usr/local/opt/gettext/lib");
-            println!("cargo:rustc-link-lib=intl");
```

Comment:
We'll probably need to add back a bunch of these dependencies once we get past the c2rust errors.